examples/fluids/newtonianwave.yaml

```diff
@@ -0,0 +1,35 @@
+problem: 'newtonian_wave'
+mu: 0 # Effectively solving Euler momentum equations
+
+dm_plex_box_faces: 40,40,1
+dm_plex_box_upper: 1,1,0.1
+dm_plex_box_lower: 0,0,0
+dm_plex_dim: 3
+bc_freestream: 4,6,3,5
+bc_slip_z: 1,2
+
+velocity_freestream: 1,1,0
+temperature_freestream: 0.25
+pressure_freestream: 71.75
+
+epicenter: 0.33,0.75,0
+amplitude: 2
+width: 0.05
+
+ts:
+  adapt_type: 'none'
+  max_steps: 400
+  dt: 3.6e-4
+  type: alpha
+  alpha_radius: 0.5
+
+implicit: true
+stab: supg
+state_var: primitive
+
+dm_mat_preallocate_skip: 0
```

Comment:
Do we still need this now that we have an analytic Jacobian?

Reply:
Nope. I think I just left it there as a vestige of a previous problem setup. Removed now.